.github/workflows/CICD.yml

```diff
@@ -122,12 +106,15 @@ jobs:
         cargo -V
         rustc -V
 
+    - name: Cross build
```

Comment:
What if instead of duplicating the steps for with cross and without cross we did something like:


```yaml
- name: Install cross
  if: ${{ matrix.job.use-cross == true }}
  run: |
    cargo install cross
    echo "CARGO_CMD=cross" >> $GITHUB_ENV
- name: Set cargo cmd
  run: echo "CARGO_CMD=cargo" >> $GITHUB_ENV
```
and then use `${{ env.CARGO_CMD }}`  to invoke cargo/cross in following commands.

Reply:
Would you prefer to `force push` or add additional commits?

Reply:
use `${{ env.CARGO_CMD }}` to invoke cargo/cross not work on Windows Powershell. (https://github.com/miles170/fd/actions/runs/3375924304/jobs/5603061216)